lib/backends/sdc/utils.js

```diff
@@ -29,6 +29,34 @@ function networkUuidToDockerId(uuid) {
     return (uuid + uuid).replace(/-/g, '');
 }
 
+function shortNetworkIdToUuid(id) {
+    var partial = '';
+
+    // consider a uuid as eight groups of four chars, plus hyphens.
+    // concatenate those groups in reverse order, jumping to the
+    // appropriate starting group.
+
+    switch (Math.ceil(id.substr(0, 32).length / 4)) {
```

Comment:
I personally find the switch easier to read than the full regex, but I think that both would make sense to the reader.